eth/protocols/snap/handler.go

```diff
@@ -319,7 +319,7 @@ func handleMessage(backend Backend, peer *Peer) error {
 				if err != nil {
 					return p2p.Send(peer.rw, StorageRangesMsg, &StorageRangesPacket{ID: req.ID})
 				}
-				var acc state.Account
+				var acc accounT.Account
```

Comment:
```suggestion
				var acc account.Account
```

Reply:
Is that intentional? That's a definite "Nope" IMO

Reply:
Yes